### ISSUE TITLE
[request] allow a wallet recovery while offline.

### ISSUE THREAD

**Issue by brianddk** [ACINQ/eclair-mobile#194]
When performing an account recovery in [v0.4.6](https://github.com/ACINQ/eclair-mobile/releases/tag/v0.4.6-MAINNET), eclair will halt and shutdown if it cannot get online.  It would be preferable to allow Eclair to continue to run offline to allow users to verify YPUB keys BTC receive addresses.

I ran into this  dilemma when researching issue https://github.com/ACINQ/eclair-mobile/issues/162 referring to the need to validate a seed.  It was suggested that the generated YPUB be compared as this would grantee the seeds match.  It is preferable to do this offline since doing a recovery online will risk channel closure if recovering to a device authorized to a different google user (and google drive account).

**Comment by pm47** [comment]
You could have imported your seed in a separate wallet (e.g. electrum) to check the ypub?

**Comment by brianddk** [comment]
@pm47 good point... wish I would have thought of that.  I was a bit curious as to what transition to another device might look like.

I'd still like the ability to recover just to a "pending sync" page or something while offline.

**Comment by ashwilly** [comment]
There was this person that helped me recover my funds from my wallet, you can PM him if you still need him on cheng66 @ protonmail . ch if you still need him to share the steps with you as well.
